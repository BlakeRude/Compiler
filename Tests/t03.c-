// C-S21
// recursive fibonacci function (two way recursion)
int fib(int n)
{
        if n<3 then  return 1;
        else return fib(n-1) + fib(n-2);
}

main()
{
        int n;

        n = fib(12);
        output(n);
        outnl();
}


