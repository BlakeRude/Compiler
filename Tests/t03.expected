Number of warnings: 0
Number of errors: 0
------------------------------------
Loading file: t03.tm
144
Bye.
